doc/install.asciidoc

```diff
@@ -398,15 +398,16 @@ See the next section for an alternative.
 Installing dependencies (system-wide Qt)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Alternatively, you can use `mkvenv.py --pyqt-mode link` to symlink your local
-PyQt/Qt install instead of installing PyQt in the virtualenv. However, unless
+Alternatively, you can use `mkvenv.py --pyqt-type link` to symlink your local
+PyQt/Qt install instead of installing PyQt in the virtualenv. You can use
+`--pyqt-version <version>` to link against a specific version. However, unless
```

Comment:
This isn't true - `--pyqt-version` is ignored with `--pyqt-type link`. Maybe it'd make sense to explain it in the section above though, perhaps even together with `--venv-dir`?